time stamp  # Time stamp of stereo pair to match against the database
